src/components/NewComps/BookFreeDemoForm/index.js

```
@@ -106,6 +114,142 @@ export default function BookFreeDemoForm({ hiddenAbsolute }) {
+              <label
+                className="form-check-label"
+                htmlFor="termsCheck"
+                style={{
+                  fontSize: "clamp(0.9rem, 1.1vw, 1rem)",
+                  lineHeight: 1.3,
+                }}
+              >
+                I accept the{" "}
+                <a
+                  href=" "
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)